x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/transform/action/GetCheckpointAction.java

```diff
@@ -105,6 +110,11 @@ public IndicesRequest indices(String... indices) {
         public boolean allowsRemoteIndices() {
             return false;
         }
+
+        @Override
+        public Task createTask(long id, String type, String action, TaskId parentTaskId, Map<String, String> headers) {
+            return new CancellableTask(id, type, action, format("get_checkpoint[%d]", id), parentTaskId, headers);
```

Comment:
I don't think it's usual to include the id in the description - I cannot see any other cancellable tasks that are constructed with the task id in the description. Or am I missing where this is commonly done?